app/content/serializers/registration.py

```diff
@@ -35,5 +38,9 @@ def get_user_info(self, obj):
             "allergy": user.allergy,
         }
 
+    def get_survey_submission(self, obj):
+        submissions = obj.submissions.filter(type=EventFormType.SURVEY)
```

Comment:
Should not user info also be used to retrieve survey submission?

Reply:
It's done in the `submissions` property which is now `get_submissions()` 😄